quantgen/R/quantile_ensemble.R

```diff
@@ -519,7 +511,7 @@ predict.quantile_ensemble = function(object, newq, s=NULL, sort=TRUE, iso=FALSE,
   # Add an all 1s matrix to newq, if we need to
   if (object$intercept) {
     a = array(NA, dim=c(n0,p+1,r))
-    for (k in 1:r) a[,,k] = cbind(rep(1,n0), newq[,,k])
+    for (k in 1:r) a[,,k] = cbind(rep(1,n0), `dim<-`(newq[,,k], apply(newq, 3, dim)[, 1]))
```

Comment:
Ditto to above, consider `drop = FALSE` as a simpler alternative?

Reply:
I don't think so - see above.

Reply:
Ok thanks.